samples/zephyr/hello-world/CMakeLists.txt

```diff
@@ -14,17 +14,6 @@ if (NOT DEFINED BOARD)
   set(BOARD frdm_k64f)
 endif()
 
-# The default top-level application configuration is prj.conf.
-# You can place additional board-specific files in boards/${BOARD}.conf,
-# and they will be merged into the configuration along with prj.conf.
-macro(set_conf_file)
-  if(EXISTS ${APPLICATION_SOURCE_DIR}/boards/${BOARD}.conf)
-    set(CONF_FILE "prj.conf ${APPLICATION_SOURCE_DIR}/boards/${BOARD}.conf")
```

Comment:
The per-board .conf file is in addition to prj.conf; it doesn't replace it. The Zephyr prj_BOARD.conf convention is for standalone conf files tat target a single board. The boards/BOARD.conf files are mix-ins. I think this patch should be dropped.

Reply:
Aha, my mistake, will fix.

Reply:
Fixed.